src/frontend/src/eth/components/send/ConvertToCkERC20.svelte

```diff
@@ -8,7 +8,7 @@
 	import ConvertETH from '$icp-eth/components/send/ConvertETH.svelte';
 	import { ckErc20HelperContractAddress } from '$icp-eth/derived/cketh.derived';
 	import { initSendContext, SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
-	import IconConvert from '$lib/components/icons/IconConvert.svelte';
+	import IconSwap from '$lib/components/icons/IconSwap.svelte';
```

Comment:
Can we use the term `Swap` in the context of ck token?

Reply:
Right! Not yet I imagine... changed to `IconCkConvert` ? WDYT?

Reply:
done